packages/demos/stories/ListPage.tsx

```diff
@@ -296,7 +297,7 @@ export default () => {
 				)}
 			</ClayManagementToolbar>
 
-			<div className="container" style={{paddingTop: 8}}>
+			<ClayLayout.Container style={{paddingTop: 8}}>
```

Comment:
In the words of my toxic teacher from elementary school: "8 what? Potatoes?". Shouldn't we explicitly say pixels here?

Reply:
I don't think its a huge issue for us to specify since in React we know that any style that is applied via integer becomes a px. I would agree with you in the case of a string, then we would need to specify.

https://reactjs.org/docs/dom-elements.html#style
> React will automatically append a “px” suffix to certain numeric inline style properties. If you want to use units other than “px”, specify the value as a string with the desired unit.